README.md

```diff
@@ -1,6 +1,6 @@
 gstat
 =====
-[![Build Status](https://travis-ci.org/edzer/gstat.png?branch=master)](https://travis-ci.org/edzer/gstat) [![CRAN](http://www.r-pkg.org/badges/version/gstat)](https://cran.rstudio.com/package=gstat) [![Downloads](http://cranlogs.r-pkg.org/badges/gstat?color=brightgreen)](http://www.r-pkg.org/pkg/gstat)
+[![Build Status](https://travis-ci.org/BenGraeler/gstat.svg?branch=master)](https://travis-ci.org/BenGraeler/gstat)
```

Comment:
This is probably something you don't want to PR?

Reply:
true, sorry